Model/Api/DiscountCodeValidation.php

```diff
@@ -241,6 +248,8 @@ public function __construct(
         $this->regionModel = $regionModel;
         $this->totalsCollector = $totalsCollector;
         $this->orderHelper = $orderHelper;
+        $this->cache = $cache ?? \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
null coalesce is not backwards compatible with PHP5

Reply:
Done.